Tests/TuistGeneratorTests/Utils/PBXGroupSorterTests.swift

```diff
@@ -0,0 +1,188 @@
+import XcodeProj
+import XCTest
+@testable import TuistGenerator
+
+class PBXGroupSorterTests: XCTestCase {
```

Comment:
This should inherit from [this class](https://github.com/tuist/tuist/blob/master/Sources/TuistSupportTesting/TestCase/TuistUnitTestCase.swift)